src/collector/Backend.cpp

```diff
@@ -84,6 +84,15 @@ void Backend::update(const BackendStat & stat)
                     std::max(m_node.get_stat().load_average, 0.01), 100.0));
     }
 
+    uint64_t last_start_old = m_stat.last_start_ts_sec * 1000000ULL + stat.last_start_ts_usec;
+    uint64_t last_start_new = stat.last_start_ts_sec * 1000000ULL + stat.last_start_ts_usec;
```

Comment:
Unifying units also should means using  common structure for timestamps everywhere.